samples/memfs/inode.go

```diff
@@ -331,6 +332,8 @@ func (in *inode) WriteAt(p []byte, off int64) (n int, err error) {
 		padding := make([]byte, newLen-len(in.contents))
 		in.contents = append(in.contents, padding...)
 		in.attrs.Size = uint64(newLen)
+		var rounded = (uint64(newLen) + uint64(in.attrs.BlockSize) - 1) & ^uint64(in.attrs.BlockSize-1)
```

Comment:
Please revert the changes to memfs, and put the tests in [statfs](https://github.com/jacobsa/fuse/blob/master/samples/statfs/statfs.go), where the existing tests for related functionality of `statfs(2)` live. You'll want to add a method called `SetBlockInfo` or similar that affects the output of `GetInodeAttributes` for `childInodeID`.

Don't forget to add tests for what happens when each value is left at its default.